garage/plotter/plotter.py

```diff
@@ -84,6 +84,9 @@ def shutdown(self):
         if not Plotter.enable:
             return
         if self._process and self._process.is_alive():
+            while not self._queue.empty():
+                self._queue.get()
+                self._queue.task_done()
```

Comment:
```
Traceback (most recent call last):
  File "/Users/jonathon/Documents/garage/garage/garage/plotter/plotter.py", line 89, in shutdown
    self._queue.task_done()
AttributeError: 'Queue' object has no attribute 'task_done'
```

Reply:
Use [JoinableQueue](https://docs.python.org/3.6/library/multiprocessing.html#multiprocessing.JoinableQueue) instead of Queue. I think python 3.6 clean up the multiprocessing a little bit.

Reply:
Done